### DISCUSSION TITLE
Offline support for Tailwind CSS documentation (PWA)

### DISCUSSION THREAD

**Discussion by kevcodez** [tailwindlabs/tailwindcss#2995] [Ideas]
It would be amazing to enable offline functionality for the tailwind docs.

When I recently had to fly for 10 1/2 hours, I wanted to lookup the tailwind docs that I had visited plenty times before. Unfortunately, since I was offline and not smart enough to download it locally before flying, I could not use the docs.

Given there are libraries to easily setup PWA functionality for Next.js (https://www.npmjs.com/package/next-pwa), this could be a good improvement to the docs 👍

**Comment by simonswiss** [comment]
Hey, 

That's a great suggestion!

Going to move this suggestion to GitHub discussions, as I am doing a little "end of the year issue cleanup" and trying to keep only real issues (bugs, incorrect documentation etc) in the issues here.

**Comment by Moberly957** [comment]
Nice collection keep up the good work.

[FaceTime PC](https://www.facetimeapp.net/facetime-for-pc/)
